x16-smc.ino

```diff
@@ -77,6 +77,7 @@ bool SYSTEM_POWERED = 0;                                // default state - Power
 int  I2C_Data[3] = {0, 0, 0};
 bool I2C_Active = false;
 char echo_byte = 0;
+bool LONGPRESS_START = 0;	// Used to let CPU know NMI has come with pwr on
```

Comment:
Even though the C++ standard guarantees that true is 1 and false is 0, I don't think it is a good practice to rely on the conversion from bool to int and vice versa like is done in the code. If nothing else, it makes the code harder to read. Consider declaring the variable as uint8_t instead.